<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QDockWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>888</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>218</width>
    <height>240</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(181, 181, 181);</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QCheckBox" name="checkBoxRoadTypeAvenue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Avenues</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRoadTypeLocalStreet">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Streets</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Extract KDE Feature</string>
    </property>
    <widget class="QPushButton" name="pushButtonExtractKDEFeature">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Extract KDE Feature</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPerturbation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Perturbation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxRotation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxExactCut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Exact Cut</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBoxRoadTypeHighway">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Highways</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRoadTypeBoulevard">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Boulevard</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
